### ISSUE TITLE
Only the first device in the list of addresses is recognized correctly

### ISSUE THREAD

**Issue by gadric** [mKeRix/room-assistant#388]
**Describe the bug**
In my configuration I have 2 devices listed. Since a couple of days I had some strange problems. I have now nailed it down to the order of the Bluetooth MAC addresses listed in the room assistant configuration. Only the first one is recognized correctly. The second one is not recognized correctly. 

This problem is observed in on INTEL NUC10, running as Home Assistant Addon. 

**To reproduce**
Change the order of the bluetooth address in the config

**Relevant logs**
Log output that correlates to the bug. You can get debug output with `room-assistant -v`.

```

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[13:41:13] INFO: Setting up Home Assistant configuration
[13:41:13] INFO: Starting room-assistant
*** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'node' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
12/7/2020, 1:41:14 PM - info - IntegrationsModule: Loading integrations: home-assistant, bluetooth-classic
12/7/2020, 1:41:14 PM - info - NestFactory: Starting Nest application...
12/7/2020, 1:41:14 PM - info - InstanceLoader: AppModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: ConfigModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: NestEmitterModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: IntegrationsModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: HttpModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: DiscoveryModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: ClusterModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: TerminusModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: BluetoothModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: ScheduleModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: EntitiesModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: HomeAssistantModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: BluetoothClassicModule dependencies initialized
12/7/2020, 1:41:14 PM - info - InstanceLoader: StatusModule dependencies initialized
12/7/2020, 1:41:14 PM - info - RoutesResolver: EntitiesController {/entities}:
12/7/2020, 1:41:14 PM - info - RouterExplorer: Mapped {/entities, GET} route
12/7/2020, 1:41:14 PM - info - RoutesResolver: StatusController {/status}:
12/7/2020, 1:41:14 PM - info - RouterExplorer: Mapped {/status, GET} route
12/7/2020, 1:41:14 PM - info - HomeAssistantService: Successfully connected to MQTT broker at mqtts://192.168.xx.yy:zzzz
12/7/2020, 1:41:14 PM - info - ConfigService: Loading configuration from /usr/lib/node_modules/room-assistant/dist/config/definitions/default.js, config/default.json, config/local.json
12/7/2020, 1:41:14 PM - info - ClusterService: Starting mDNS advertisements and discovery
12/7/2020, 1:41:14 PM - info - NestApplication: Nest application successfully started
12/7/2020, 1:41:15 PM - info - ClusterService: Added XXXXXXXXXXX:6425 to the cluster with id bedroom
12/7/2020, 1:41:16 PM - info - HomeAssistantService: Device tracker requires manual setup in Home Assistant with topic: room-assistant/device_tracker/bluetooth-classic-XXXXXXXXXXXXXXXXXX-tracker/state
12/7/2020, 1:41:57 PM - info - HomeAssistantService: Device tracker requires manual setup in Home Assistant with topic: room-assistant/device_tracker/bluetooth-classic-YYYYYYYYYYYYYYYYYY-tracker/state
```

**Relevant configuration**
Paste the relevant parts of your configuration below.

```
global:
  cluster:
    autoDiscovery: false
    port: 6425
    peerAddresses:
      - 'Peer Address 1'
      - 'Peer Address 2'
  instanceName: floor
  integrations:
    - homeAssistant
    - bluetoothClassic
homeAssistant:
  mqttUrl: 'mqtts://<ip>:<port>'
  mqttOptions:
    username: 
    password:
    rejectUnauthorized: false
bluetoothClassic:
  hciDeviceId: 0
  interval: 10
  scanTimeLimit: 5
  timeoutCycles: 5
  minRssi: -30
  addresses:
    - 'MAC Address 1'
    - 'MAC Address 2'
```

**Expected behavior**
Both Devices should be correctly recognized

**Environment**
 - room-assistant version: 2.12.0
 - installation type: Home Assistant Addon
 - hardware: Intel NUC10
 - OS: Debian 10 Linux

**Additional context**

**Comment by mKeRix** [comment]
Are your other instances receiving requests to inquire for the second device or none at all (should be visible when running room-assistant in verbose mode)? Does it make a difference which instance is currently the leader? The MAC addresses to inquire will be taken from the leader instance config.

Apart from that, small side note: the `cluster` block of your config needs to be indented one level less (= top-level) to function properly. :)

**Comment by gadric** [comment]
`Are your other instances receiving requests to inquire for the second device or none at all (should be visible when running room-assistant in verbose mode)? Does it make a difference which instance is currently the leader? The MAC addresses to inquire will be taken from the leader instance config. `

How can I start the verbose mode on Home Assistant Addon? Docu says to run room-assistant —verbose. But on home assistant Addon I can’t run this ssh. And if I execute shell in the docker container directly, it reports: 

```
12/8/2020, 1:41:01 PM - info - NestApplication: Nest application successfully started
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::6415
    at Server.setupListenHandle [as _listen2] (net.js:1313:16)
    at listenInCluster (net.js:1361:12)
    at Server.listen (net.js:1447:7)
    at ExpressAdapter.listen (/usr/lib/node_modules/room-assistant/node_modules/@nestjs/platform-express/adapters/express-adapter.js:45:32)
    at NestApplication.listen (/usr/lib/node_modules/room-assistant/node_modules/@nestjs/core/nest-application.js:146:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
Emitted 'error' event on Server instance at:
    at emitErrorNT (net.js:1340:8)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'listen',
  address: '::',
  port: 6415
}
```
It there another way to get to verbose mode?


```
Apart from that, small side note: the cluster block of your config needs to be indented one level less (= top-level) to function properly. :)
```

Thanks for the hint :) I have changed it :) I am wondering why it never reported an error :)

**Comment by mKeRix** [comment]
Unfortunately the Home Assistant add-on does not provide a straightforward way of switching into verbose mode yet - I had hoped that you could run at least one of the other instances in verbose mode and observe if the RSSI requests are received there.

Let me just double check so that I am sure I completely understand the problem:
- you have a cluster of multiple room-assistant instances, including one that is running as an add-on
- all instances have these two MAC addresses configured for BT Classic
- the other instances report correctly for both MACs, the add-on instance only reports for MAC 1
Am I describing your issue correctly?

> I am wondering why it never reported an error :)

room-assistant does not validate the config yet, it just takes what it gets. That is on my wishlist for the future, to improve user experience. :)

**Comment by gadric** [comment]
Yes, your understanding of the problem is correct. 

I am done some further digging, and found out that it is somehow related to the BT of the NUC. I have connected an external adapter and it worked without any problems. 

What I do not understand is that, with the onboard BT, only the first one is recognized. I would understand if none of the devices are recognized. But only the first one?!

**Comment by gadric** [comment]
```UPDATE 2020-12-13```

I just fixed the error with [incompatibility with docker 20.10](https://alerts.home-assistant.io/#docker_2010.markdown). This seems also fixed this problem that only the first device is recognized. 

So in this setup, this seems to work without any problems.

**Comment by gadric** [comment]
`UPDATE 2020-12-18`: 

Since Home Assistant fixed the problems with docker 20.10, I upgraded again. I am facing this problem. I am currently try to debug again. 

Is room assistant executing `hcitool -i hci0 cc <mac address> && hcitool -i hci0 rssi <mac address>` to get rssi? and in a cluster I assume that it gathers all rssi value and set the room to the instance which has the lowest rssi value right?
